h2o-automl/src/main/java/ai/h2o/automl/AutoMLBuildSpec.java

```diff
@@ -63,20 +57,28 @@ public AutoMLBuildControl() {
     public boolean keep_cross_validation_models = false;
     public boolean keep_cross_validation_fold_assignment = false;
     public String export_checkpoints_dir = null;
+
+    public AutoMLBuildControl() {
+      stopping_criteria = new AutoMLStoppingCriteria();
+
+      // reasonable defaults:
+      stopping_criteria.set_max_models(0);
```

Comment:
Is -1 or 0 more common as a replacement for None/NULL for an integer-valued parameter on the backend?  I have seen both... (which I guess means we can choose whichever one is "better"?).

Reply:
@ledell I didn't change that code, just moved it down: usually in Java, attributes are declared before constructor, I was just applying this "norm" here.

Regarding the -1/0 debate... there's no rule written in stone. -1 is commonly used for "unset", and when there's no ambiguity like here, I tend to use 0 for "unlimited". Maybe it deserves a tiny comment next to it.